src/enrich/typesense-index.ts

```diff
@@ -59,11 +59,11 @@ export const enrichTypesense: Enrichment<{}, { record: SingleRecord; foundTopics
       extraTopics[`topic_${k}`] = v;
     }
 
-    let plaintext = '';
-    const keywordsFile = join(api.files, 'keywords.txt');
-    if (existsSync(keywordsFile)) {
-      plaintext = await readFile(keywordsFile, 'utf-8');
-    }
+    // let plaintext = '';
+    // const keywordsFile = join(api.files, 'keywords.txt');
+    // if (existsSync(keywordsFile)) {
+    //   plaintext = await readFile(keywordsFile, 'utf-8');
+    // }
```

Comment:
We can look at this again in the future, it was a way to have scripts that extract plaintext (into the `keywords.txt`) for indexing - but it made the output 2-3x larger!